### PR TITLE
feat: show when a replay is still ongoing

### PR DESCRIPTION
**Pull request by pauldambra**
People periodically ask for "live" session replay

Other than ingestion lag we don't make it clear that often they _are_ watching the live replay

Let's show them when we received data in the last 5 minutes

Screenshot 2024-09-23 at 13 11 35
Screenshot 2024-09-23 at 13 33 44
Screenshot 2024-09-23 at 13 34 22

### CODE REVIEW

File: posthog/session_recordings/queries/session_recording_list_from_filters.py [hunk 1]
```diff
@@ -60,7 +60,10 @@ class SessionRecordingListFromFilters:
             (duration - active_seconds) as inactive_seconds,
             sum(s.console_log_count) as console_log_count,
             sum(s.console_warn_count) as console_warn_count,
-            sum(s.console_error_count) as console_error_count
+            sum(s.console_error_count) as console_error_count,
+            -- Check if the most recent _timestamp is within five minutes of the current time
+            -- proxy for a live session
+            (max(s._timestamp) >= now() - INTERVAL 5 MINUTE) as ongoing
```
Comment (daibhin) [inline posthog/session_recordings/queries/session_recording_list_from_filters.py:66]: The inactivity period (30 minutes by default) feels like a more exact interval, but 5 minutes will probably be more accurate so let's go with that for now

File: frontend/src/scenes/session-recordings/playlist/SessionRecordingPreview.tsx [hunk 2]
```diff
@@ -153,7 +153,15 @@ function PinnedIndicator(): JSX.Element | null {
     )
 }
 
-function ViewedIndicator(): JSX.Element {
+function RecordingOngoingIndicator(): JSX.Element {
+    return (
+        <Tooltip title="This recording is still ongoing - we received data within the last 5 minutes.">
+            <IconLive />
```
Comment (daibhin) [inline frontend/src/scenes/session-recordings/playlist/SessionRecordingPreview.tsx:159]: This feels like it should be live somewhat:
pulse

You can achieve this with some simple custom Tailwind:
```suggestion
            <IconLive className="animate-[pulse_1s_ease-out_infinite] text-primary-3000" />
```

**Review by daibhin — APPROVED** [review]

File: posthog/session_recordings/queries/session_recording_list_from_filters.py (hunk 1 above)
Comment (pauldambra) [reply posthog/session_recordings/queries/session_recording_list_from_filters.py:66]: David you are now officially better at making things pulse than ChatGPT 🤣

File: frontend/src/scenes/session-recordings/playlist/SessionRecordingPreview.tsx (hunk 2 above)
Comment (pauldambra) [reply frontend/src/scenes/session-recordings/playlist/SessionRecordingPreview.tsx]: You are now officially better than ChatGPT at making things pulse

Comment (daibhin) [reply frontend/src/scenes/session-recordings/playlist/SessionRecordingPreview.tsx]: Test me on my knowledge of the ancient mayan civilisation next... 👀

**Pull request merged**